test/Mono.Linker.Tests/Tests/TypeNameParserTests.cs

```diff
@@ -1,125 +0,0 @@
-﻿using NUnit.Framework;
-
-namespace Mono.Linker.Tests
-{
-	[TestFixture]
-	public class TypeNameParserTests
```

Comment:
We should try to adapt these test cases - even if the code is very different the tests themselves seem to be valid.

Reply:
I adapted these.